build.gradle

```diff
@@ -79,6 +79,10 @@ uploadArchives {
             id 'vnnv'
             name 'Victor Nikiforov'
           }
+          developer {
+              id 'ghost81'
+              name 'Iliya Ivanov'
```

Comment:
Hi Iliya, thanks for the PR.  Did you really want to be added into the developer section?

Victor built the original and I refactored it to then release.  In general I don't want to encourage the use of @Author annotations and <developer> sections as we then start getting into the areas of contributor agreements and all that stuff.  I'd rather people just give their contributions freely rather than with conditions etc.

Cheers, Rob.